ports/geojson-cpp/vcpkg.json

```
@@ -0,0 +1,9 @@
+{
+  "name": "geojson-cpp",
+  "version-string": "0.5.1",
```

```suggestion
  "version-semver": "0.5.1",
```
This filed might be `version-semver`. Please see https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions